### PR TITLE
AO3-4763 – PseudsController Strong Params

### PR DESCRIPTION
**Pull request by kamilogorek**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4763

### CODE REVIEW

**Comment by kamilogorek** [comment]
One of the features that was failing is in `features/collections/icon.feature` and not `features/other_a/pseud*`.

File: app/controllers/pseuds_controller.rb [hunk 1]
```diff
@@ -146,4 +146,14 @@ def destroy
 
     redirect_to(user_pseuds_url(@user))
   end
+
+  private
+
+  def pseud_params
+    params.require(:pseud).permit(
+      :user_id, :name, :description, :is_default, :icon, :delete_icon, :icon_file_name,
```
Comment (zz9pzza) [inline app/controllers/pseuds_controller.rb]: I am not convinced that the following need to be allowed though:

user_id,  :icon_file_name, :icon_content_type, :icon_file_size, :icon_alt_text, :delta

Comment (sarken) [inline app/controllers/pseuds_controller.rb]: It does seem to work without `:icon_alt_text`, both in the sense that the pseud is created/updated *and* the specified alt text is saved, but I'm a little confused _why_ it works. Do you know, james? (I checked and taking out something like `:name` does cause a problem.)

Comment (zz9pzza) [inline app/controllers/pseuds_controller.rb]: All I commented on is that it is not in the form :)

Comment (sarken) [inline app/controllers/pseuds_controller.rb]: But it is -- right here: https://github.com/otwcode/otwarchive/blob/master/app/views/pseuds/_pseuds_form.html.erb#L48

Comment (zz9pzza) [inline app/controllers/pseuds_controller.rb]: You are completely correct. I must of been thinking about the icon_file_name. sorry ( have to look at how it works now )

**Pull request merged**